server.go

```diff
@@ -1053,16 +1053,17 @@ func SaveMultipartFile(fh *multipart.FileHeader, path string) (err error) {
 		return
 	}
 
+	var tmpfile string
 	var ok bool
 	if ff, ok = f.(*os.File); ok {
 		// Windows can't rename files that are opened.
 		if err = f.Close(); err != nil {
 			return
 		}
-
+		tmpfile = ff.Name()
```

Comment:
Is it possible to add unit tests?

Reply:
not possible, the case happened when save file to another logic volumn.